contrib/babelfishpg_tsql/sql/upgrades/babelfishpg_tsql--3.1.0--3.2.0.sql

```diff
@@ -421,8 +421,18 @@ CALL sys.babelfish_drop_deprecated_object('view', 'sys', 'sysprocesses_deprecate
 CALL sys.babelfish_drop_deprecated_object('view', 'sys', 'dm_exec_sessions_deprecated_in_3_2_0');
 CALL sys.babelfish_drop_deprecated_object('function', 'sys', 'tsql_stat_get_activity_deprecated_in_3_2_0');
 
+CREATE OR REPLACE FUNCTION sys.bbf_get_context_info()
+RETURNS sys.VARBINARY(128) AS 'babelfishpg_tsql', 'bbf_get_context_info' LANGUAGE C STABLE;
+
 CREATE OR REPLACE FUNCTION sys.context_info()
-RETURNS sys.VARBINARY(128) AS 'babelfishpg_tsql' LANGUAGE C STABLE;
+RETURNS sys.VARBINARY(128)
```

Comment:
I wonder do we need this 2level structure?

Reply:
I don't think you can specify both AS 'definition' and AS 'obj_file', 'link_symbol' in the same function definition.